src/ExpressionEvaluator/Core/Source/FunctionResolver/CSharp/CSharpFunctionResolver.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.ExpressionEvaluator;
+using Microsoft.VisualStudio.Debugger.FunctionResolution;
+
+namespace Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator
+{
+    [DkmReportNonFatalWatsonException(ExcludeExceptionType = typeof(NotImplementedException)), DkmContinueCorruptingException]
```

Comment:
The Dispatcher can return E_NOTIMPL if no implementations are found when making a call on a Dkm object.  I wouldn't remove existing NFW attributes in the EC/RP, but this one may not be necessary.